mbed_tools_lib/logging.py

```diff
@@ -0,0 +1,33 @@
+"""Helpers for logging errors according to severity of the exception."""
+import logging
+from typing import Any, Optional
+
+
+def log_exception(logger: Optional[Any],
+                  exception: Optional[Exception]) -> None:
```

Comment:
Are these really optional?

Reply:
Yeah, that doesn't seem to make sense to me, why would you want to log and exception without a logger or an exception?